sass/ding_header_opening_hours.scss

```diff
@@ -16,93 +14,109 @@ $gray-transparent-bg:     rgba( 0, 0, 0, 0.7);
-  src:  url('../fonts/icomoon.eot?bueezz#iefix') format('embedded-opentype'),
-  url('../fonts/icomoon.ttf?bueezz') format('truetype'),
-  url('../fonts/icomoon.woff?bueezz') format('woff'),
-  url('../fonts/icomoon.svg?bueezz#icomoon') format('svg');
-  font-weight: normal;
-  font-style: normal;
-}
-
-[class^="icon-"], [class*=" icon-"] {
-  /* use !important to prevent issues with browser extensions that change fonts */
-  font-family: 'icomoon' !important;
-  speak: none;
-  font-style: normal;
-  font-weight: normal;
-  font-variant: normal;
-  text-transform: none;
-  line-height: 1;
-
-  /* Better Font Rendering =========== */
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
 .icon-clock {
-  font-size: 24px;
+  font-size: 55px;
+  margin-top: -10px;
+  padding-left: 10px;
   color: $color-primary;
   float: right;
-}
-.icon-clock:before {
-  content: "\e920";
-}
 
+  &:before {
+    content: "\e90e";
+    font-family: 'icomoon';
+  }
+}
 
 .site-header .topbar .topbar-inner {
   overflow: visible;
   min-height: 89px;
 }
 
+#block-ding-header-opening-hours-ding-header-opened-today {
+  .ohs-wrapper {
+    min-height: 48px;
+
+    div.oh-wrapper:first-child:nth-last-child(1) {
+      line-height: 40px;
+    }
+  }
+}
+
 .not-logged-in {
-   .pane-opened-libraries {
-    @include zen-grid-item(3.8, 6.8);
-    @include small-medium() {
-      display: none;
-     }
+  .pane-opened-libraries {
+    // @include zen-grid-item(3.8, 6.8);
+    @include span-columns(2 of 6);
+    position: absolute;
+    right: 200px;
+  }
+
+  @include media($tablet) {
+    // @include zen-grid-item(3.8, 4.6);
+    .pane-opened-libraries {
+      @include span-columns(3 of 7);
+      top: 74px;
+      z-index: 10;
+      margin-left: 105px;
+    }
+    .pane-content #block-ding-header-opening-hours-ding-header-opened-today-all {
+      .ohs-wrapper .oh-row .left {
+        width: 11em;
+      }
+    }
   }
 }
 
 .logged-in {
   .pane-opened-libraries {
-    @include zen-grid-item(3.2, 3);
+    // @include zen-grid-item(3.2, 3);
+    @include span-columns(2 of 6);
+    position: absolute;
+    left: 200px;
+  }
 
-    @include small-medium() {
-      display: none;
+  @include media($notebook) {
+    // @include zen-grid-item(3.8, 4.6);
+    .pane-opened-libraries {
+      @include span-columns(3 of 8);
+      margin-left: -85px;
     }
+  }
 
-    @media #{$tablet} {
-      @include zen-grid-item(3.8, 4.6);
-
-      .pane-content #block-ding-header-opening-hours-ding-header-opened-today-all {
-        .ohs-wrapper .oh-row .left {
-          width: 11em;
-        }
+  @include media($tablet) {
+    // @include zen-grid-item(3.8, 4.6);
+    .pane-opened-libraries {
+      @include span-columns(3 of 7);
+      top: 74px;
+      z-index: 10;
+      margin-left: 30px;
+    }
+    .pane-content #block-ding-header-opening-hours-ding-header-opened-today-all {
+      .ohs-wrapper .oh-row .left {

```

Comment:
Consider sass best practices for maximum 3 selectors for style. Also, use nested approach